python/GafferSceneUI/SetEditor.py

```diff
@@ -171,6 +175,12 @@ def __dragBegin( self, widget, event ) :
 			if column == self.__setMembersColumn :
 				GafferUI.Pointer.setCurrent( "paths" )
 				return IECore.StringVectorData( self.__getSetMembers().paths() )
+			elif column == self.__includedSetMembersColumn :
+				GafferUI.Pointer.setCurrent( "paths" )
+				return IECore.StringVectorData( self.__getIncludedSetMembers().paths() )
+			elif column == self.__excludedSetMembersColumn :
+				GafferUI.Pointer.setCurrent( "paths" )
+				return IECore.StringVectorData( self.__getExcludedSetMembers().paths() )
```

Comment:
This only works if the thing you're dragging happens to have been selected already - see below :

https://github.com/GafferHQ/gaffer/assets/1133871/494789ad-0f7b-4005-abbe-dc58437a3399

I think the original code for `self.__setMembersColumn` is working because clicking on the members column changes the selection before the drag begins, but the inclusions/exclusions columns can't do that. Perhaps we should change all the drag code to follow this pattern?

- If the dragged-upon thing is selected, then drag all the selected things.
- If the dragged-upon thing is not selected, just drag the dragged-upon thing.

Reply:
Sounds good to me! I've adjusted the overall drag behaviour in 79754ea.